tla/raft_spec/MCraft.tla

```diff
@@ -0,0 +1,49 @@
+
+---------- MODULE MCraft ----------
+EXTENDS ccfraft, TLC
+
+PossibleServer_mc == {NodeOne, NodeTwo, NodeThree, NodeFour, NodeFive}
+InitialServer_mc == {NodeOne, NodeTwo, NodeThree}
+tmp == [s \in PossibleServer_mc |-> Follower]
+InitialConfig_mc == [tmp EXCEPT ![NodeFour] = Pending,
+                                ![NodeFive] = Pending]
```

Comment:
The first node launched in a CCF network actually does a `force_become_leader` step, rather than waiting for an initial election. So in practice the initial state for us is `{Primary, Follower, Follower, Pending, Pending}`. Not sure how useful this is, but it could remove some searching of equivalent states early in the execution (ie - once the state is stable after the initial election, we don't care whether the winner was `NodeOne`, `NodeTwo`, or `NodeThree`, so we can avoid searching all of them by just starting _from_ that point with the winner as `NodeOne`)?

Reply:
That is a very good point. I'll keep the current model as is but will add a PR later with a slightly modified initialization for node states (we can also already start with a partial log in my opinion to make things interesting). Thanks for the idea!